### ISSUE TITLE
Bootloop

### ISSUE THREAD

**Issue by EdwardMastah** [Magisk-Modules-Repo/PIXELARITY#5]
I've encountered some problem with the flash file. After rebooting for the first time it takes unlimited amount of time that it will never reach the lockscreen. Knowing this, it is maybe a bootloop. And this is my problem.

Ps. I'm using AOSP Extended rom

**Comment by okaminoseishin** [comment]
I'm also experiencing this bug, also on the AEX. It's does not matter which combination of installation options you chose, boot loop guaranteed.
**AEX**: 6.7
**Magisk**: 20.0
**PIXELARITY**: 3.8
**Device**: Xiaomi Redmi 5 Plus (vince)

**Comment by Zyphs** [comment]
Same issue here
**OnePlus 3**
**LinageOS**: 16.0
**Magisk**: 20.0
**PIXELARITY**: 3.8

*edit
I fixed it just re-flashed LinageOS and Magisk and downloaded the moduel again
i made sure to not install any gapps

**Comment by okaminoseishin** [comment]
> *edit
> I fixed it just re-flashed LinageOS and Magisk and downloaded the moduel again
> i made sure to not install any gapps

No, it's device or OS-specific workaround. With only OS, Magisk and PIXELARITY result still the same.
Plus, not using GAPPS may not be applicable to all ROMs. AEX, for example, has two variants - with and without them, also it isn't applicable to those who want to fall into complete Google ecosystem.

**Comment by mhamzas** [comment]
Same issue
LEECO Le Pro 3 (ZL1)
AICP : 15
Magisk: 23.0
PIXELARITY: 3.8
